### ISSUE TITLE
PAR Recessive inheritance

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7491]
**Preferred term label:**
PAR Recessive inheritance

**Synonyms**
Pseudo Autosomal Region Recessive inheritance

**Definition (free text, please give PubMed ID)**
 Gonosomal inheritance that is recessive and in which biallelic males and females both manifest a disease phenotype

**Parent term (use hpo.jax.org/app)**
Gonosomal inheritance HP:0010985

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034341
